pkg/btree/multi.go

```diff
@@ -97,6 +99,52 @@ func (m *LinkedMetaPage) MarshalMetadata(bm encoding.BinaryMarshaler) error {
 	return m.SetMetadata(buf)
 }
 
+func (m *LinkedMetaPage) NextNOffsets() ([]uint64, error) {
+	offsets := make([]uint64, N)
+	currentOffset := m.offset
+
+	for i := 0; i < N; i++ {
+		if _, err := m.rws.Seek(int64(currentOffset)+12, io.SeekStart); err != nil {
```

Comment:
Ahh I mean I don't think this code should iterate through offsets.

As in, this entire function should be

```
Seek(currentOffset + 12)
for i := 0; i < N; i++ {
  binary.Read(rws, binary.LittleEndian, &offset[i]);
}
```

It should only read from the current meta page.